src/ymfm_opl.cpp

```diff
@@ -1715,9 +1715,14 @@ uint8_t ymf278b::read_status()
 
 uint8_t ymf278b::read_data_pcm()
 {
-	// write to FM
-	if (bitfield(m_address, 9) != 0)
-		return m_pcm.read(m_address & 0xff);
+	// read from PCM
+	if (bitfield(m_address, 9) != 0) {
```

Comment:
Minor: please follow the spacing/bracing conventions in the rest of the file.

Reply:
Done.